crates/aptos/README.md

```diff
@@ -581,3 +581,130 @@ Additionally, profiles can replace addresses in the function id.
 ```bash
 aptos move run --function-id default::Message::set_message --args string:hello!
 ```
+
+## Genesis Ceremonies
+
+The `aptos` tool supports bootstrapping new blockchains through what is known as a genesis ceremony. The output of the genesis ceremony is the output of move instructions that prepares a blockchain for online operation. The input consists of:
+
+* A set of validators and their configuration
+* The initial set of Move modules, known as a framework
+* A unique `ChainId` (u8) that distinguishes this from other deployments
+* For test chains, there also exists an account that manages the minting of TestCoin
+
+## Generating Genesis
+
+* The genesis organizer constructs a `Layout` and distributes it.
+* The genesis organizer prepares the Aptos framework's bytecode and distributes it.
+* Each participant generates their `ValidatorConfiguration` and distributes it.
+* Each participant generates a `genesis.blob` from the resulting contributions
+* The genesis organizer executes the `genesis.blob` to derive the initial waypoint and distributes it.
```

Comment:
waypoint can be generated by participant too, maybe here we say organizer will publish genesis.blob and waypoint, participant will validate they got same result? (e.g. sha256)

Reply:
Should discuss this -- if our viewpoint is that the waypoint is going to be used by participants from the one they generate, we might as well make the waypoint optional and reduce operational burden.

Reply:
well right now genesis.blob and waypoint are generate in one command, so they would have both anyway